airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss

```diff
@@ -0,0 +1,25 @@
+@use "../../scss/colors";
+
+@keyframes spinning {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.spinner {
+  animation: spinning 1000ms linear infinite normal forwards;
+}
+
+.centerDot {
+  fill: colors.$blue;
+}
+
+:export {
+  // Export colors to be used in SVG gradients
+  gradientStart: colors.$blue;
+  gradientStop: colors.$blue-transparent;
+}
```

Comment:
Does these need to be re-named and re-exported here? Seems simpler to just import directly from `./scss/export.module.scss` in `CircleLoader.tsx`?

Reply:
Hmm great my comment that I explained that in somehow vanished :thinking:

The idea is that `export.module.scss` is deprecated and meant to be removed onces we switched everything away from styled-components to SASS. This is a (very rare) place that will need though to still have access to those colors in TS even with styled-component removed, so I decided to excplicitally export them here, to not later block the removal of the export.module.scss.

Reply:
I see, didn't realize that `export.module.scss` is deprecated along with `theme.ts` 👍